project-base/storefront/components/Pages/Customer/OrderDetailContent.tsx

```diff
@@ -276,6 +288,23 @@ export const OrderDetailContent: FC<OrderDetailContentProps> = ({ order, breadcr
                     </div>
                 )}
             </Webline>
+
+            {!!orderForPrefillingUuid && (
+                <Popup onCloseCallback={() => setOrderForPrefillingUuid(undefined)}>
```

Comment:
Let's import this Popup component as `dynamic`.

Reply:
Done